tests/unit/adapters/application-test.js

```diff
@@ -116,7 +116,7 @@ test('#findRelated can be called with optional type for the resource', function
   let service = this.container.lookup('service:people');
   let supervisor = this.container.lookup('model:employee').create({
     type: 'supervisors',
```

Comment:
This should be `people`, see the example resource prototypes at https://github.com/pixelhandler/ember-jsonapi-resources/blob/master/tests/helpers/resources.js#L40-L48